### PR TITLE
Add FORCE_CREATE_MANIFEST to Validation Process

### PR DESCRIPTION
**Pull request by MaribelleHGomez**
Description about what this pull request does.

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### New Features
- Add logic for modifying validation process when files are expected to be redacted
- Validation process still error out as it did previously, but when FORCE_CREATE_MANIFEST True it will proceed to create a manifest output instead of halt the entire validation process

### CODE REVIEW

File: scripts/validate.py [hunk 1]
```diff
@@ -51,6 +52,11 @@ def run(global_config):
             "Expecting non-empty IGNORED_FILES. Please check if ignored_files_manifest.csv is configured correctly!!!"
         )
 
+    logger.info(
+        "If validation job is run with FORCE_CREATE_MANIFEST True: errors from missing objects are to be expected due to redaction of records in the data release following the current run"
+    )
+    force_create_manifest = global_config.get("force_create_manifest")
```
Comment (BinamB) [inline scripts/validate.py:58]: hah you're gonna hate me. Can you change these to all caps since its a global variable :grimacing:

Comment (MaribelleHGomez) [reply scripts/validate.py:58]: all of those var from the config?

File: scripts/validate.py [hunk 2]
```diff
@@ -51,6 +52,11 @@ def run(global_config):
             "Expecting non-empty IGNORED_FILES. Please check if ignored_files_manifest.csv is configured correctly!!!"
         )
 
+    logger.info(
```
Comment (BinamB) [inline scripts/validate.py:55]: lol another one. Could you add this log under an `if FORCE_CREATE_MANIFEST` . So if true then let user know with the logs

File: scripts/validate.py [hunk 3]
```diff
@@ -51,6 +52,11 @@ def run(global_config):
...
+        "If validation job is run with FORCE_CREATE_MANIFEST True: errors from missing objects are to be expected due to redaction of records in the data release following the current run"
+    )
+    FORCE_CREATE_MANIFEST = global_config.get("FORCE_CREATE_MANIFEST")
```
Comment (BinamB) [inline scripts/validate.py:58]: Can you do `get("FORCE_CREATE_MANIFEST", False)` we need a default value if value is not provided

**Review by BinamB — APPROVED** [review]

**Pull request merged**